<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monte Carlo Simulation - Estimating Pi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="5">
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thread Four</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QProgressBar" name="simBar4">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thread Three</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pi Estimate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="estimateLCD">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thread One</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QProgressBar" name="simBar1">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QProgressBar" name="simBar3">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="analyzeButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QProgressBar" name="simBar2">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thread Two</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLCDNumber" name="accuracyLCD">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Confidence Interval</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Settings</string>
        </property>
        <widget class="QSpinBox" name="setDiameter">
         <property name="geometry">
          <rect>
           <x>91</x>
           <y>20</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Circle Diameter</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="setIts">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="activeThread1">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="activeThread2">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGraphicsView" name="simView1">
        <property name="cacheMode">
         <set>QGraphicsView::CacheBackground</set>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QGraphicsView" name="simView2">
        <property name="transformationAnchor">
         <enum>QGraphicsView::NoAnchor</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="activeThread3">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="activeThread4">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGraphicsView" name="simView3">
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QGraphicsView" name="simView4">
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
